apps/nova/src/app/[locale]/(auth)/not-wishlist/page.tsx

```
@@ -0,0 +1,33 @@
+import { createAdminClient } from '@tuturuuu/supabase/next/server';
+import { getTranslations } from 'next-intl/server';
+import { redirect } from 'next/navigation';
+
+export default async function NotWhitelistedPage() {
+  const t = await getTranslations();
+  const user = await getCurrentUser();
+
+  if (!user?.email) redirect('/login');
+  console.log(user.email, ' emal');
+  const adminSb = await createAdminClient();
+  const {} = await adminSb
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)